packages/utils/src/logger/logger.ts

```diff
@@ -27,8 +27,8 @@ export const setupConfigLogger = () => {
 }
 
 export abstract class Logger {
-  static getLogger(category?: string, useDebug = true): ConfigLogger | DebugLogger {
-    return useDebug ? new DebugLogger(category.replace('.', ':')) : new ConfigLogger(category)
+  static getLogger(category?: string, useDebug = true): Logger {
+    return useDebug ? new DebugLogger(category) : new ConfigLogger(category)
```

Comment:
Why removing the `replace`?
Is that because `:` in `debug` library is just a convention?
https://github.com/visionmedia/debug#conventions

Reply:
Indeed, `debug` uses `:`

Reply:
So we need to keep the replace, as I replaced all instance of `:` to `.` in the logger category (log4js convention)